src/components/TextInput/index.jsx

```diff
@@ -1,7 +1,34 @@
-import React from "react";
+import React, { useState } from "react";
+import styles from "./styles/styles.module.scss";
 
-const TextInput = () => {
-  return <p>text input</p>;
+const TextInput = ({ label, minLength, isRequired = true }) => {
+  let [errorText, setErrorText] = useState("");
+
+  const handleInputValue = (event) => {
+    const textLength = event.target.value.length;
+
+    if (textLength === 0) {
+      setErrorText("Can't be empty.");
+    } else if (textLength < { minLength }) {
```

Comment:
Would you elaborate this part a bit? Why you have added `{}` around `minLength`? 🤔 
```jsx
    } else if (textLength < { minLength }) {
```

Reply:
that was my mistake
thanks for mention